rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/ListDeadLetterSourceQueuesDirectives.scala

```diff
@@ -0,0 +1,72 @@
+package org.elasticmq.rest.sqs
+
+import org.elasticmq.actor.reply._
+import org.elasticmq.msg.ListDeadLetterSourceQueues
+import org.elasticmq.rest.sqs.Action.{ListDeadLetterSourceQueues => ListDeadLetterSourceQueuesAction}
+import org.elasticmq.rest.sqs.Constants._
+import org.elasticmq.rest.sqs.directives.ElasticMQDirectives
+import org.elasticmq.rest.sqs.model.RequestPayload
+import spray.json.DefaultJsonProtocol._
+import spray.json.RootJsonFormat
+
+import scala.xml.Elem
+
+trait ListDeadLetterSourceQueuesDirectives { this: ElasticMQDirectives with QueueURLModule with ResponseMarshaller =>
+  def listDeadLetterSourceQueues(p: RequestPayload)(implicit marshallerDependencies: MarshallerDependencies) = {
+    p.action(ListDeadLetterSourceQueuesAction) {
+      rootPath {
```

Comment:
It seems that removing `rootPath` directive solves the issue with sdk test. Not sure why this is happening :shrug: